### PR TITLE
doc: Update all docs for v1 release

### PR DESCRIPTION
**Pull request by luanpotter**
I will take a look at #1101 after this and update any necessary default values + docs on a followup.
I think this is good to merge, the only remaining TODO is to add some more info on troubleshooting about gapless loop, but hoping someone can pick that up after this lands.
Hopefully after these two things we can release an RC2

### CODE REVIEW

File: feature_parity_table.md [hunk 1]
```diff
@@ -2,10 +2,29 @@
 
 Not every feature is available on every platform yet. Use this table to keep track of our work and progress, and please help if you want :)
 
-Note: LLM means Low Latency Mode.
+## Note on Linux Support
+
+Our main goal in your current roadmap is to support Linux. This is the current status of Desktop for us:
```
Comment (Sundin) [inline feature_parity_table.md:7]: Should probably be "Our main goal in **the** current roadmap" or something like that!

File: getting_started.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+### Release Mode
+
+The release mode is controlling what happens when the playback ends. There are 3 options:
+
+1. `.stop`: just stops the playback but keep all associated resources.
+1. `.release` (default): releases all resources associated with this player, equivalent to calling the `release` method.
+1. `.loop`: starts over after completion, looping over and over again.
+
+**Note**: you can control exactly what happens when the playback ends using the `onAudioComplete` stream (see Streams below).
```
Comment (Sundin) [inline getting_started.md:131]: Should be `onPlayerComplete` according to the code, if I'm not mistaken.

File: feature_parity_table.md [hunk 3]
```diff
@@ -2,10 +2,29 @@
...
+
+* [DONE] Add platform interface, refactor API so native implementations are cleaner
+* [DONE] Federate plugin and provide structure to add multi-platform support
+* [DONE] Add macOS support through Darwin (existing)
+* [DONE] (thanks @azchohfi) Add Windows support
+* [IN PROGRESS] Add Linux support
+
+If you would like to assist us on this final step, please reach our on our Discord server so we can coordinate efforts.
```
Comment (Sundin) [inline feature_parity_table.md:15]: Where do I find this Discord server?

File: getting_started.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+**Note**: despite our best efforts, some native SDK implementations that we use spam a lot of log messages that we currently haven't figured out how to conform to this configuration (specially noticeable on Android). If you would like to contribute with a PR, they are more than welcome!
+
+## Audio Context
+
+An Audio Context is a (mostly mobile-specific) set of secondary, platform-specific aspects of audio playback, typically related to how the act of playing audio interacts with other features of the device. In most cases, you do not need to change this.
+
+The Audio Context configuration can be set globally via:
+
+```dart
+  AudioPlayer.global.setGlobalAudioContext(config);
```
Comment (Sundin) [inline getting_started.md:172]: Can't find this, on the other hand I found this way that seem to work: `GlobalPlatformInterface.instance.setGlobalAudioContext(config);`

Comment (Sundin) [reply getting_started.md:172]: (See https://github.com/bluefireteam/audioplayers/issues/1101 for full context)

File: feature_parity_table.md (hunk 3 above)
Comment (luanpotter) [inline feature_parity_table.md:15]:
```suggestion
If you would like to assist us on this final step, please reach our on our [Discord](https://discord.gg/pxrBmy4) server so we can coordinate efforts.
```

Comment (luanpotter) [reply feature_parity_table.md:15]: good point!

File: feature_parity_table.md (hunk 1 above)
Comment (luanpotter) [inline feature_parity_table.md:7]:
```suggestion
Our main goal in the current roadmap is to support Linux. This is the current status of Desktop for us:
```

File: getting_started.md (hunk 2 above)
Comment (luanpotter) [inline getting_started.md:131]:
```suggestion
**Note**: you can control exactly what happens when the playback ends using the `onPlayerComplete` stream (see Streams below).
```

File: getting_started.md (hunk 4 above)
Comment (luanpotter) [reply getting_started.md:172]: That is the way on rc1. I am adding a shortcut `AudioPlayer.global` on this very PR

**Review by spydon — APPROVED** [review]

**Review by erickzanardo — APPROVED** [review]

**Pull request merged**